src/amo/pages/TagPage/index.js

```diff
@@ -0,0 +1,88 @@
+/* @flow */
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import HeadLinks from 'amo/components/HeadLinks';
+import Page from 'amo/components/Page';
+import Search from 'amo/components/Search';
+import { DEFAULT_TAG_SORT } from 'amo/constants';
+import translate from 'amo/i18n/translate';
+import {
+  convertFiltersToQueryParams,
+  convertQueryParamsToFilters,
+  fixFiltersFromLocation,
+} from 'amo/searchUtils';
+import type { SearchFilters } from 'amo/api/search';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'amo/types/i18n';
+import type { ReactRouterMatchType } from 'amo/types/router';
+
+type Props = {|
+  match: {|
+    ...ReactRouterMatchType,
+    params: {| tag: string, visibleAddonType: string |},
```

Comment:
Is `visibleAddonType` a URL parameter? I thought the URL pattern was only `/tag/:tag/` and if that's the case, `visibleAddonType` should be removed here.

Reply:
https://github.com/mozilla/addons-frontend/pull/10825/commits/0922f516e66a0345d790d53e67f58ab1e6d03115